db/migrate/20190422184900_rename_organisation_id_to_organization_id.rb

```diff
@@ -1,5 +0,0 @@
-class RenameOrganisationIdToOrganizationId < ActiveRecord::Migration[5.2]
-  def change
```

Comment:
never remove the migration
if it has no use or is invalid - either refactor it or comment the code so it'd be empty